packages/fx-core/src/plugins/resource/simpleauth/utils/common.ts

```diff
@@ -38,6 +50,21 @@ export class Utils {
     );
   }
 
+  public static generateBicepFiles(templateFilePath: string, pluginsContext: any): string {
```

Comment:
try to find another name for `pluginsContext` parameter? Since it's not PluginContext type.

Reply:
Maybe put this function to a common place. Seems different resource plugin will share same code?

Reply:
Moved to *packages\fx-core\src\common\tools.ts*

Reply:
> try to find another name for `pluginsContext` parameter? Since it's not PluginContext type.

Changed to `context`.

Reply:
Is this method a generic handlebars compile method?

Reply:
Yes. Moved the method to *packages\fx-core\src\common\tools.ts*